.postgraphilerc.js

```diff
@@ -1,6 +1,7 @@
 require('dotenv').config()
 module.exports = {
   options: {
+    watch: true,
```

Comment:
Note: this is still not working properly, not sure why. If you create a new lookup table, then try and view it, it will fail until you stop and restart the pg server. But it's broken on develop too, so need to fix this.